examples/combobox/combobox-datepicker.html

```diff
@@ -0,0 +1,781 @@
+
+          <div class="dialog-message" aria-live="polite"></div>
+
+        </div>
+      </div>
+    </div>
+    <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+  </section>
+
+  <section>
+    <h2>Accessibility Features</h2>
+    <p>There are three main features of the combobox date picker example:</p>
+    <ul>
+      <li>Textbox: Holds the value of the selected date and can open the date picker dialog box.</li>
+      <li>Choose Date Button: Opens the datepicker dialog box.</li>
+      <li>Date Picker Dialog Box: Displays a grid of dates for the user to select from and provides additional buttons to change the month and year of dates shown in the grid.</li>
+    </ul>
+
+    <p>The <code>combobox</code>, <code>textbox</code> and <code>button</code> all have the same accessible name.</p>
+
+
+    <h3>Date Textbox</h3>
+
+    <ul>
+      <li>Contains the date value.</li>
+      <li>Opens and closes the date picker dialog box through keyboard, click and touch events.</li>
+      <li>A down arrow icon in the <code>textbox</code> provides a visual affordance that the <kbd>down arrow</kbd> key opens the date picker dialog box.</li>
+      <li>A description provides information on the date format to assistive technologies.</li>
+    </ul>
+
+    <h3>Choose Date Button</h3>
+
+    <ul>
+      <li>The down arrow icon is a button to open and close the date picker dialog box through click events.</li>
+      <li>The arrow points down when the dialog box is closed and it points up when the dialog box is open.</li>
+      <li>Excluded from the <kbd>tab</kbd> sequence of the page (eg. <code>tabindex=-1</code>), since the user can use the <kbd>down arrow</kbd> or <kbd>enter</kbd> key to open the date picker dialog box from the <code>textbox</code>.</li>
+    </ul>
+
+    <h3>Date Picker Dialog</h3>
+
+    <ul>
+      <li>The date picker dialog is a modal dialog box for selecting a date from a grid of dates for a particular month and year.</li>
+      <li>Additional buttons and keyboard commands are used to change month and year.</li>
+      <li>A live region announces changes in the month and year.</li>
+      <li>A live region announces the use of cursor keys when a date in the grid of dates gets focus.  The message is also visible at the bottom of the dialog box to provide the same information to keyboard users.</li>
+      <li>The dialog is opened through keyboard commands in the textbox or by clicking on the choose date button.</li>
+      <li>High contrast support for focus and hover styling of the controls in the dialog box use the CSS border property:
+        <ul>
+          <li>When a button or date receives focus a border is added.</li>
+          <li>When hovering over a button or date with a pointing device a border is added.</li>
+          <li>By default buttons and dates do not have a border, padding is used as a placeholder for the added border for focus and hover styling.</li>
```

Comment:
Add comma, and split into 2 sentences.

```suggestion
          <li>By default, buttons and dates do not have a border. Padding is used as a placeholder for the added border for focus and hover styling.</li>
```